components/TodoList.js

```diff
@@ -19,14 +31,51 @@ const HighPriorityTitle = styled(PriorityTitle)`
 
 const TodoItem = styled.li`
   display: flex;
-  align-items: center;
-  margin-bottom: 4px;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-bottom: 12px;
+  border-radius: 10px;
+  padding: 10px;
+  background-color: #fff;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+`;
+
+const TodoText = styled.span`
+  text-decoration: ${(props) => (props.completed ? 'line-through' : 'none')};
 `;
 
-const TodoText = styled.span``;
+const ButtonContainer = styled.div`
+  display: flex;
+  margin-top: 5px;
+`;
 
 const TodoButton = styled.button`
-  margin-left: 8px;
+  margin: 0 5px;
+  background-color: ${(props) =>
+    props.clear ? 'red' : props.add ? 'green' : props.edit ? 'orange' : 'transparent'};
```

Comment:
this is a bit hard to read, you could use a better version where the different styling is separated: 
```js
margin: 0 5px;
${(props) => props.clear && css`
  background-color: 'red';
`
}

${(props) => props.add && css`
  background-color: 'green';
`
}
...
```
This helps the reader to distinguish between the different button variants.

Reply:
the "css" must be imported from styled-components